### PR TITLE
GITC-641: fix broken save cart as collection modal image

### LINKED ISSUE #10019: Image missing in Save Cart as Collection modal
**To reproduce**
- Add Grant to cart
- Click save to collection
- View missing image in modal.

**Acceptance Criteria**
Image should be displayed under the title of the modal.

[Jira Link](https://gitcoin.atlassian.net/browse/GITC-641?atlOrigin=eyJpIjoiNWQ2ZGUwOTQ0ZmZmNGU4ZDkxMzBjZTVmMTBmYmIyNTAiLCJwIjoiaiJ9)

**Screenshot**
image

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Image should be displayed under the title of the modal.

##### Refers/Fixes

closes #10019 

##### Testing

https://share.vidyard.com/watch/qdFBjUs4hiAiNEUuXdL1J4?

### CODE REVIEW

**Review by ksolo — APPROVED** [review]

**Review by schultztimothy — APPROVED** [review]
LGTM

File: app/assets/v2/js/grants/create-collection-modal.js [hunk 1]
```diff
@@ -48,15 +45,12 @@ Vue.component('create-collection-modal', {
     },
     createCollection: async function() {
       const csrfmiddlewaretoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
-      const checkedOut = CartData.loadCheckedOut();
-      const cart = (checkedOut.length > 0 ? checkedOut : CartData.loadCart());
-      const grantIds = cart.map(grant => grant.grant_id);
```
Comment (gdixon) [inline app/assets/v2/js/grants/create-collection-modal.js:53]: `grantIds` was an array here but `this.grantIds` is a string, have you tested the collection creation flow?

File: app/assets/v2/js/grants/create-collection-modal.js [hunk 2]
```diff
@@ -6,21 +6,19 @@ Vue.component('create-collection-modal', {
       collectionTitle: '',
       collectionDescription: '',
       collections: [],
+      grantIds: '',
```
Comment (gdixon) [inline app/assets/v2/js/grants/create-collection-modal.js:9]:
```suggestion
      grantIds: [],
```

File: app/assets/v2/js/grants/create-collection-modal.js [hunk 3]
```diff
@@ -6,21 +6,19 @@ Vue.component('create-collection-modal', {
...
       showCreateCollection: false
     };
   },
   mounted: function() {
-    const checkoutData = CartData.loadCheckedOut();
+    const checkedOut = CartData.loadCheckedOut();
+    const cart = (checkedOut.length > 0 ? checkedOut : CartData.loadCart());
 
-    let grant_ids = '';
+    this.grantIds = cart.map(grant => grant.grant_id).join(',');
```
Comment (gdixon) [inline app/assets/v2/js/grants/create-collection-modal.js:19]:
```suggestion
    this.grantIds = cart.map(grant => grant.grant_id);
```

File: app/assets/v2/js/grants/create-collection-modal.js [hunk 4]
```diff
@@ -6,21 +6,19 @@ Vue.component('create-collection-modal', {
...
 
-    for (let i = 0; i < checkoutData.length; i++) {
-      grant_ids = grant_ids + checkoutData[i]['grant_id'] + ',';
-    }
-
-    this.imgURL = '/dynamic/grants_cart_thumb/' + document.contxt['github_handle'] + '/' + grant_ids;
+    this.imgURL = '/dynamic/grants_cart_thumb/' + document.contxt['github_handle'] + '/' + this.grantIds;
```
Comment (gdixon) [inline app/assets/v2/js/grants/create-collection-modal.js:21]:
```suggestion
    this.imgURL = '/dynamic/grants_cart_thumb/' + document.contxt['github_handle'] + '/' + this.grantIds.join(',');
```

File: app/assets/v2/js/grants/create-collection-modal.js (hunk 1 above)
Comment (chibie) [reply app/assets/v2/js/grants/create-collection-modal.js:53]: oops! nice catch @gdixon 

thanks

**Review by gdixon — APPROVED** [review]
LGTM! 🚀

**Pull request merged**